engine_eval/zanzibar/authz-openfga-ex/go.mod

```diff
@@ -5,7 +5,155 @@ go 1.19
 require (
 	github.com/golang/glog v1.0.0
 	github.com/openfga/go-sdk v0.1.1
+	github.com/testcontainers/testcontainers-go v0.16.0
 	gopkg.in/yaml.v2 v2.4.0
+
+)
+
+require (
+	github.com/AlecAivazis/survey/v2 v2.3.6 // indirect
+	github.com/Azure/go-ansiterm v0.0.0-20210617225240-d185dfc1b5a1 // indirect
```

Comment:
Not sure, why it require all such imports

Reply:
you can try `go mod tidy` to see if it helps

Reply:
nope, already tried, doesn't help. I asked in the testcontainers [slack](https://testcontainers.slack.com/archives/CBWBNR76G/p1671032409726519) about it, let's see what they say about it :)